api/base/settings/defaults.py

```diff
@@ -328,3 +328,5 @@
         'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
     },
 }
+
+SLOAN_ID = 'sloan_id'
```

Comment:
this name is unclear -- looking at other settings, we often specify `_COOKIE_NAME` to make it clear what it's for